go/libkb/globals.go

```diff
@@ -353,6 +353,16 @@ func (g *GlobalContext) ConfigureConfig() error {
 func (g *GlobalContext) ConfigReload() error {
 	err := g.ConfigureConfig()
 	g.ConfigureUpdaterConfig()
+	g.ConfigureDeviceCloneState()
+	return err
+}
+
+func (g *GlobalContext) ConfigureDeviceCloneState() error {
+	c := NewDeviceCloneStateJSONFile(g)
+	err := c.Load(false)
```

Comment:
why open this here?  why not just open it whenever you need to access it?

Reply:
It was just so I could put it in the ConfigReload method which gets called in a bunch of places. It seemed like the kind of thing that might be (or might one day) be called to naively and hopefully clean things up that might be in a bad state. But yes I agree that it doesn't at all seem necessary. still want me to remove it from here?

Reply:
i'll remove it. that doesn't really seem like a compelling enough reason to have it here.